bom/pom.xml

```diff
@@ -31,7 +31,7 @@
     <artemis.image.name>enmasseproject/activemq-artemis:2.2.0-4</artemis.image.name>
     <assertj-core.version>3.11.1</assertj-core.version>
     <californium.version>2.0.0-M16</californium.version>
-    <dispatch-router.image.name>quay.io/enmasse/qdrouterd-base:1.7.0</dispatch-router.image.name>
+    <dispatch-router.image.name>quay.io/enmasse/qdrouterd-base:1.8.0</dispatch-router.image.name>
```

Comment:
sadly, we cannot simply update the image to a new minor version as Dispatch Router is a pre-req for Hono, i.e. we need to file a CQ for this.

Reply:
Ok, I will remove it and simply state in the guide that when taking the Service Bus option you have to deploy a 1.8